client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -196,7 +193,9 @@ const CopySiteItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
 			href={ copySiteHref }
 			onClick={ () => {
 				recordTracks( 'calypso_sites_dashboard_site_action_copy_site_click' );
-				startSiteCopy();
+				if ( onClick ) {
+					onClick();
+				}
```

Comment:
We can simplify the code like this:
```suggestion
				onClick?.();
```